model/kops_metadata.go

```diff
@@ -49,6 +50,7 @@ type KopsMetadataRequestedState struct {
 	NodeInstanceType   string `json:"NodeInstanceType,omitempty"`
 	NodeMinCount       int64  `json:"NodeMinCount,omitempty"`
 	NodeMaxCount       int64  `json:"NodeMaxCount,omitempty"`
+	Networking         string `json:"networking,omitempty"`
```

Comment:
```suggestion
	Networking         string `json:"Networking,omitempty"`
```
To match capitalization with the rest.

Reply:
Resolved. thank you